.rubocop.yml

```diff
@@ -200,6 +200,15 @@ RSpec/DescribeClass:
     - 'spec/features/**/*.rb'
     - 'modules/*/spec/features/**/*.rb'
 
+# Allow number HTTP status codes in specs
+RSpecRails/HttpStatus:
+  Enabled: false
+
+# have_http_status does not exist on MockResponse
+# using it would yield "expected a response object, but an instance of Rack::MockResponse was received"
+RSpecRails/HaveHttpStatus:
+  Enabled: false
+
```

Comment:
I see. I have an experiment (one may call it a hack) to get it working with `Rack::MockResponse`. I'll provide a PR about it so that we can rely on `have_http_status` anywhere.

Reply:
Should not be necessary anymore with https://github.com/opf/openproject/pull/15951.

Reply:
Thanks for fixing this 💙